ui/src/settings.vue

```diff
@@ -109,7 +108,7 @@ export default {
     border-bottom: 1px solid #eee;
 }
 
-/deep/ .container h5 {
+.container h5 {
```

Comment:
should I change it back to `/deep/ .container h5 { `?

Reply:
this PR has nothing to do with settings, right? Please don't change it.

Reply:
yes! while merging the updates I mistakenly did it. Sorry.